src/main.ts

```diff
@@ -293,6 +294,13 @@ export default class ImporterPlugin extends Plugin {
 				helpPermalink: 'import/notion',
 				formatDescription: 'Export your Notion workspace to HTML format.',
 			},
+			'notion-api': {
+				name: 'Notion (via API)',
```

Comment:
change to `Notion (API)`

Reply:
Done. https://github.com/obsidianmd/obsidian-importer/pull/444/commits/f73ac30206c9670ed465074c1dbade96d3196662